src/components/PostProcessedApiSpec/PostProcessedApiSpec.tsx

```diff
@@ -26,10 +27,48 @@ const PostProcessedApiSpec: React.FC = () => {
       ) : error ? (
         <S.ErrorLabel>{error.message}</S.ErrorLabel>
       ) : (
-        data && <SwaggerUI spec={YAML.parse(data)} plugins={[TableOfContentsPlugin, DynamicServersPlugin]} />
+        data && <SwaggerUI spec={parseSpec(YAML.parse(data))} plugins={[TableOfContentsPlugin, DynamicServersPlugin]} />
```

Comment:
I'm thinking that parsing will take some time and it would probably be better to do it in a `useMemo` only when the `data` state has changes